content/1.7/back-office/international/locations/countries/add-edit-country.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Add edit country
+weight: 1
+---
+
+# Add edit country
+
+When adding or editing a country, the following fields / drop-down / toggles are displayed:
+
+ - **Country** - This field is mandatory. When trying to save without filling this field, the following error message is displayed: "The field name is required at least in _default language_.". This field can be translated: When several languages are installed, a drop-down next to the field is displayed with the default language selected. It allows choosing in which language the field is displayed and customizing it according to the language. Disabled languages are displayed in the drop-down.
+The following tooltip is displayed: "Country name - Invalid characters: <>;=#{}".
+
+ - **ISO code** - This is a mandatory field, when trying to proceed without enetering this field, the following error message is displayed: "The iso_code field is required.". If the field input is invalid, the following error message is displayed: "The iso_code field is invalid." The following tooltip is displayed: "Two -- or three -- letter ISO code (e.g. "us" for United States).". If you try to add a country with an already existing ISO, the following error will show: "This ISO code already exists.You cannot create two countries with the same ISO code."
+
+ - **Call prefix** - This field is mandatory. The following tooltip is displayed: "Internation call prefix, (e.g. 1 for United States)". When trying to save without filling this field, the following error message is displayed: "The Call prefix field is required.". 
+ 
+ - **Default currency** - This drop-down allows to select a currency from the ones that are enabled on the shop. By default, the drop-down is set to: "Default store currency".
+ 
+ - **Zone** - This drop-down allows selecting a zone. All the zones are listed (both enabled and disabled ones). The following tooltip is displayed: "Geographical region.".
+ 
+ - **Does it need Zip/Postal code?** - This toggle indicates whether a user living in this country must give a zip code or not when signing up to the shop. The toggle can be put on "yes" or "no". By default, it is set to "yes". 
+
+ - **Zip/Postal code format** - This field is mandatory.
+The following help text is displayed: "Indicate the format of the postal code: use L for a letter, N for a number, and C for the country's ISO 3166-1 alpha-2 code. For example, NNNNN for the United States, France, Poland and many other; LNNNNLLL for Argentina, etc. If you do not want PrestaShop to verify the postal code for this country, leave it blank."
```

Comment:
```suggestion
The following help text is displayed: "Indicate the format of the postal code: use L for a letter, N for a number, and C for the country's ISO 3166-1 alpha-2 code. For example, NNNNN for the United States, France, Poland and many other; LNNNNLLL for Argentina, etc. If you do not want PrestaShop to verify the postal code for this country, leave it blank." If the field input is invalid, the following error message is displayed: "The zip_code_format field is invalid.".
```

Reply:
We can save without filling this field and without any error message. Is it mandatory?